lte/cloud/go/services/lte/obsidian/models/enodeb_state_swaggergen.go

```diff
@@ -70,6 +70,9 @@ type EnodebState struct {
 
 	// Time at which the state was reported in ms
 	TimeReported uint64 `json:"time_reported,omitempty"`
+
+	// ues connected
+	UesConnected int32 `json:"ues_connected,omitempty"`
```

Comment:
Will this value get set in a forthcoming PR? Right now it's always gonna be zero